<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aloha_server</class>
 <widget class="QDialog" name="Aloha_server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtQQ_Server</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>698</width>
      <height>498</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>700</width>
       <height>500</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>702</x>
     <y>1</y>
     <width>296</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>main_logo.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>299</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>299</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>用户姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>用户寸照：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectPictureBtn">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>70</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新增用户</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>  用户寸照</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queryIDBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查询用户</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="queryIDLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>注销用户</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="logoutIDLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入用户账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="userAccountLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>用户账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userAccountLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入用户账号</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
